includes/reader-revenue/woocommerce/class-woocommerce-cover-fees.php

```diff
@@ -145,16 +146,18 @@ private static function should_apply_fee( $data ) {
 
 	/**
 	 * Render the "cover fees" input for Stripe.
+	 *
+	 * @param string $payment_gateway The slug for the payment gateway rendering these payment fields.
 	 */
-	public static function render_stripe_input() {
-		if ( ! self::should_allow_covering_fees() ) {
+	public static function render_stripe_input( $payment_gateway ) {
```

Comment:
I understand WCPay fees are via Stripe, but perhaps we should rename this method since we are not only supporting Stripe gateway anymore? WDYT?

Reply:
That's a good point. ba8d687a updates this method name plus a couple of others, some inline comments, and the order note added to donation orders when the donor opts in to covering fees, to remove mentions of Stripe and clarify that the option applies to all Newspack-supported payment gateways (currently just Stripe and WooPayments) .